uuid.go

```diff
@@ -38,7 +38,7 @@ func (u *UUID) DeepCopyInto(out *UUID) {
 
 // Equal returns true if itself is equal to other.
 func (u UUID) Equal(other UUID) bool {
-	return uuid.Equal(u.UUID, other.UUID)
+	return u == other
```

Comment:
Is it working have you tested that?

Reply:
It should work, cf https://github.com/gofrs/uuid/releases/tag/v3.0.0